app/src/main/java/ca/bc/gov/bchealth/compose/component/AnnouncementBannerUI.kt

```diff
@@ -0,0 +1,182 @@
+package ca.bc.gov.bchealth.compose.component
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.BoxWithConstraints
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.material.Card
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.layoutId
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.constraintlayout.compose.ConstraintLayout
+import androidx.constraintlayout.compose.ConstraintSet
+import androidx.constraintlayout.compose.Dimension
+import androidx.constraintlayout.compose.Visibility
+import ca.bc.gov.bchealth.R
+import ca.bc.gov.bchealth.compose.BasePreview
+import ca.bc.gov.bchealth.compose.theme.HealthGatewayTheme
+import ca.bc.gov.bchealth.compose.theme.bannerBackgroundBlue
+import ca.bc.gov.bchealth.compose.theme.blue
+
+private const val bannerIconId = "bannerIconId"
+private const val bannerTitleId = "bannerTitleId"
+private const val bannerArrowId = "bannerArrowId"
+private const val bannerBodyId = "bannerBodyId"
+private const val buttonLearnModeId = "buttonLearnMoreId"
+private const val buttonDismissId = "buttonDismissId"
+
+@Composable
+fun AnnouncementBannerUI(
```

Comment:
Is this the communication Banner? There is already a compose version for that: BannerUI

Reply:
Since when I started this it was not in develop & the new UI is made using new theme and components it is more streamlined. so I will eliminate the BannerUI in future PR.